epub33/rs/index.html

```diff
@@ -319,7 +319,7 @@ <h3>Unique Identifier</h3>
 				<p>Reading Systems SHOULD NOT depend on the <code>unique-identifier</code> attribute being unique to one
 					and only one EPUB Publication. Determining whether two EPUB Publications with the same Unique
 					Identifier represent different versions of the same publication, or different publications, may
-					require inspecting other metadata, such as the titles or authors.</p>
+					require inspecting other metadata, such as the last modification date, titles, and authors.</p>
 			</section>
```

Comment:
Just to be picky:-)... shouldn't this be ", or" instead of ", and"?

(I must admit I am often mixed up by the usage of "and" vs. "or" in these types of lists in English...)

Reply:
It's a list of possible choices that can all be used, not an exclusive list of options in which you pick only one. The original wording using "or" makes it sound like you pick one.

Reply:
I trust you on that one:-)